tests/TileDB.CSharp.Test/ArrayTest.cs

```diff
@@ -125,6 +125,7 @@ public void TestSparseArray()
         }
 
         [TestMethod]
+        [Ignore("Causes CI failures.")]
```

Comment:
I debugged this test locally on Ubuntu this morning and found the only reason it fails is because the first entry in `string[] fragments` is just a literal letter `o`, while the rest of the items in the array are fragment names. Is this not the case for windows? Just curious, wondering what's going on that seems to only impact Ubuntu.

In any case this is deprecated, so I think we could indicate that in the Ignore message and add a test for the new consolidate API (which doesn't need the `fragments[]` arg so that wouldn't be an issue)

Reply:
When I debugged it the strings were entirely made of invalid data. In any case this is a Core issue and let's ignore this test until it gets fixed to reduce the noise.